src/content/docs/en/guides/integrations-guide/lit.mdx

```diff
@@ -126,6 +126,34 @@ import { MyElement } from '../components/my-element.js';
 Lit requires browser globals such as `HTMLElement` and `customElements` to be present. For this reason the Lit renderer shims the server with these globals so Lit can run. You *might* run into libraries that work incorrectly because of this.
 :::
 
+### Experimental Decorators
+
+To use experimental decorators in Lit,, add the following to your `tsconfig.json` file:
+
+```json title="tsconfig.json" add={3}
+{
+  "compilerOptions": {
+    "experimentalDecorators": true,
+  }
+}
+```
+
+This allows you to use [Lit's experimental decorators](https://lit.dev/docs/components/decorators/) in your Lit component:
```

Comment:
``````suggestion
To use [experimental decorators in Lit](https://lit.dev/docs/components/decorators/) , add the following to your `tsconfig.json` file:

```json title="tsconfig.json" add={3}
{
  "compilerOptions": {
    "experimentalDecorators": true,
  }
}
```

This allows you to use experimental decorators such as `@customElement` and `@state` to register a custom element and define a state property in your Lit component:
``````

What do you think about this? Probably the link should now be at the first mention, rather than the second?

Reply:
Great! Thank you